src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h

```diff
@@ -958,6 +958,13 @@ The actual algorithm is described in
 		}
 	};
 
+	class xSTIR_RelativeDifferencePrior3DF : public stir::RelativeDifferencePrior < float > {
+	public:
+		void only2D(int only) {
+			only_2D = only != 0;
```

Comment:
Perhaps more methods need to be exposed by this accessor class (i.e. kappa, gamma and epsilon)?

Reply:
public members in the STIR class are inherited. It would make sense to override `set/get_kappa_sptr` could be overridden by SIRF versions that use `SIRFImageData`. No need for `gamma` and `epsilon` as far as I can see

Reply:
Right, as inherited these don't need exposed.